src/pointer-analysis/value_set.h

```diff
@@ -276,6 +277,7 @@ class value_sett
     const namespacet &ns) const;
 
   /// Appears to be unimplemented.
+  DEPRECATED(SINCE(2019, 05, 22, "Unimplemented"))
   expr_sett &get(const irep_idt &identifier, const std::string &suffix);
```

Comment:
I think we can just remove what isn't even implemented.

Reply:
I don't know. In test-gen there were cases of cpp files used instead of the cbmc ones which could add new implementation not existing in cbmc